manifests/params.pp

```diff
@@ -164,4 +166,13 @@
   $certificate_whitelist      = [ ]
   # change to this to only allow access by the puppet master by default:
   #$certificate_whitelist      = [ $::servername ]
+
+  # Get the parameter name for the database connection pool tuning
+  if $puppetdb_version in ['latest','present'] or versioncmp($puppetdb_version, '4.0.0') >= 0 {
+    $database_max_pool_size_setting_name = 'maximum-pool-size'
+  } elsif versioncmp($puppetdb_version, '3.2.0') >= 0 {
```

Comment:
I'm absolutely happy to do that - but I have no way of testing anything less than 3.2 for myself.